plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/AccumuloMetadata.java

```diff
@@ -238,7 +238,12 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
     }
 
     @Override
-    public Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session, ConnectorInsertTableHandle insertHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics)
+    public Optional<ConnectorOutputMetadata> finishInsert(
```

Comment:
Iceberg connector still uses the deprecated finishInsert method. Did you skip the change intentionally?

Reply:
nit: Blackhole is also not modified.
I'm guessing you are making these changes to eventually remove the deprecated method.

Reply:
Iceberg requires more changes, because finishInsert is used internally by create table flow

Reply:
and yes, there is still a bit more work tbd to remove the old overload